org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/engine/connected/ConnectedEngineFacade.java

```diff
@@ -620,19 +557,36 @@ public ConnectedEngineFacade setNotificationsDisabled(boolean value) {
 
   public void downloadServerIssues(String projectKey, @Nullable String branchName, IProgressMonitor monitor) {
```

Comment:
Not sure this method is still used (it is already not used on master)

Reply:
It is used in [AnalyzeConnectedProjectJob](https://github.com/SonarSource/sonarlint-eclipse/blob/3946e656b667db38dfbca4150895dc93b5c9dc6c/org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/jobs/AnalyzeConnectedProjectJob.java#L79)
Seems Eclipse is broken and can not reliably find call sites anymore :(